frappe/core/doctype/deleted_document/deleted_document_list.js

```diff
@@ -0,0 +1,41 @@
+frappe.listview_settings["Deleted Document"] = {
+	onload: function (doclist) {
+		const action = () => {
+			const selected_docs = doclist.get_checked_items();
+			if (selected_docs.length > 0) {
+				let docnames = selected_docs.map((doc) => {
+					return doc.name;
+				});
+				frappe.call({
+					method: "frappe.core.doctype.deleted_document.deleted_document.bulk_restore",
+					args: { "docnames": docnames },
+					callback: function (r) {
+						if (r.message) {
+							function body(docnames) {
+								const href = "<li><a href='/desk#Form/Deleted Document/%(0)s'>%(0)s</a></li>";
+								const html = docnames.map(docname => { return repl(href, {'0': docname}) });
+								return "<br><ul>" + html.join("");
+							}
+							function message(title, docnames) {
+								return (docnames.length > 0) ? __(title) + body(docnames) + "</ul>": "";
+							}
+
+							let { restored, invalid, failed } = r.message;
+							let restored_summary = message("Documents restored successfully", restored);
```

Comment:
```suggestion
							let restored_summary = message(__("Documents restored successfully"), restored);
```

https://frappeframework.com/docs/user/en/translations#1-literal-strings

Do it for all titles and messages in this PR.